src/components/panes/configure-panes/keycode.tsx

```diff
@@ -147,9 +149,21 @@ export const KeycodePane: FC = () => {
   const disableFastRemap = useAppSelector(getDisableFastRemap);
   const selectedKeyDefinitions = useAppSelector(getSelectedKeyDefinitions);
   const {basicKeyToByte} = useAppSelector(getBasicKeyToByte);
+  const [numMacros, setNumMacros] = useState(16);
```

Comment:
```suggestion
  const numMacros = useAppSelector(getMacroCount);
```

There should be a selector to get the macro count so you shouldn't need to make another API request for it.